charts/topolvm/README.md

```diff
@@ -171,6 +171,7 @@ You need to configure kube-scheduler to use topolvm-scheduler extender by referr
 | securityContext.runAsUser | int | `10000` | Specify runAsUser. |
 | snapshot.enabled | bool | `true` | Turn on the snapshot feature. |
 | storageClasses | list | `[{"name":"topolvm-provisioner","storageClass":{"additionalParameters":{},"allowVolumeExpansion":true,"annotations":{},"fsType":"xfs","isDefaultClass":false,"reclaimPolicy":null,"volumeBindingMode":"WaitForFirstConsumer"}}]` | Whether to create storageclass(s) ref: https://kubernetes.io/docs/concepts/storage/storage-classes/ |
+| useLegacy | bool | `false` | If true, the legacy plugin name and legacy custome resource group is used(topolvm.cybozu.com). |
```

Comment:
Sorry, this is an auto-generated file...
It will be fixed if you resolve the following typo.
https://github.com/topolvm/topolvm/pull/539#discussion_r979744693